### PR TITLE
Bug 1243807 - Update requirements so that we stop getting CPOW errors.

### PR DESCRIPTION
**Pull request by sydvicious**
Update marionette requirements to avoid CPOW errors. r?maja_zf

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,8 +7,8 @@
 PACKAGE_VERSION = '0.5'
 
 deps = [
-    'marionette-client == 2.1.0',
-    'marionette-driver == 1.2.0',
+    'marionette-client == 2.2.0',
+    'marionette-driver == 1.3.0',
     'mozlog == 3.1',
     'manifestparser == 1.1',
     'firefox-puppeteer >= 3.1.0, <4.0.0',
```
Comment (mjzffr) [inline setup.py]: there's a left-over 3.1.0 here

File: requirements.txt [hunk 2]
```diff
@@ -15,10 +15,10 @@ mozrunner==6.11
 moztest==0.7
 mozversion==1.4
 wptserve==1.3.0
-marionette-client == 2.1.0
-marionette-driver == 1.2.0
-marionette-transport == 1.1.0
-firefox-puppeteer==3.1.0
+marionette-client == 2.2.0
+marionette-driver == 1.3.0
+marionette-transport == 1.2.0
```
Comment (mjzffr) [inline requirements.txt]: Remove marionette-transport (see "Removing marionette-transport package" on tools-marionette)

**Comment by mjzffr** [comment]
You could also bump wptserve to 1.4.0 if https://bugzilla.mozilla.org/show_bug.cgi?id=1247014 is taken care of soon. (They usually respond pretty quickly.)

**Comment by sydvicious** [comment]
I'll bump wptserve when that bug lands,

**Comment by mjzffr** [comment]
Merged
https://github.com/mjzffr/firefox-media-tests/commit/751e363ab214244ff49b74c03454135c38db01da

**Pull request closed without merging**